templating/twig_extension_repository.rst

```diff
@@ -40,7 +40,7 @@ Then, define the extension class as a service and tag it with the
 
         # app/config/services.yml
         services:
-            twig_extension.intl:
+            twig_extensions.intl:
```

Comment:
I guess `app.` is better since there may be other bundles who try the same that we do here.